### PR TITLE
docs: testing a fix for text in codeblocks

### PR DESCRIPTION
**Pull request by brnhensley**
These two sections were rendering text in codeblocks and code as text, I'm testing if changing the indentation will correct it

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

File: src/content/docs/logs/logs-context/net-configure-logs-context-all.mdx [hunk 1]
```diff
@@ -319,38 +319,38 @@ You can use the [Apache log4net version 2.0.8 or higher](https://logging.apache.
...
+      ```
 
-    ```yml
-    logs:
-      - name: application-log
-	      file: C:\logs\log4netExample.log.json # Path to a single log file
-    ```
-    
-    #[Documentation for using the infrastructure agent forwarder](https://docs.newrelic.com/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent/#file)
+      #[Documentation for using the infrastructure agent forwarder](https://docs.newrelic.com/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent/#file)
```
Comment (barbnewrelic) [inline src/content/docs/logs/logs-context/net-configure-logs-context-all.mdx:353]: I don't think this line should begin with #. Maybe reword to:
See the [infrastructure agent forwarder documentation]/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent/#file).

File: src/content/docs/logs/logs-context/net-configure-logs-context-all.mdx [hunk 2]
```diff
@@ -567,11 +567,11 @@ You can use our [Serilog](https://serilog.net/) extension to link to your log da
...
-      - name: application-log
-        file: C:\logs\log4netExample.log.json # Path to a single log file
-    ```
+      ```yml
+      logs:
+        - name: application-log
+          file: C:\logs\log4netExample.log.json # Path to a single log file
+      ```
     
 #[Documentation for using the infrastructure agent forwarder](https://docs.newrelic.com/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent/#file)
```
Comment (barbnewrelic) [inline src/content/docs/logs/logs-context/net-configure-logs-context-all.mdx:576]: same comment as before re: starting this line with #

**Review by barbnewrelic — APPROVED** [review]
As far as I could tell, the code looks right. I just had a comment about the line after code blocks that links to infra doc. Hope this helps! Otherwise looks great! Let me know if you still want to test, or if you want me to merge this in. No rush. Thanks again, @brnhensley  !

File: src/content/docs/logs/logs-context/net-configure-logs-context-all.mdx (hunk 2 above)
Comment (brnhensley) [reply src/content/docs/logs/logs-context/net-configure-logs-context-all.mdx]: I corrected those too Barb

**Comment by brnhensley** [comment]
The @barbnewrelic It looks like this edit worked and is all good now, thanks for the feedback!

**Comment by barbnewrelic** [comment]
Thanks again for all your contributions to docs!
